dev/storage.h

```diff
@@ -142,33 +142,82 @@ namespace sqlite_orm {
                 perform_void_exec(db, ss.str());
             }
 #endif
+            inline void add_generated_cols(std::vector<table_xinfo*>& columnsToAdd,
+                                           std::vector<table_xinfo>& storageTableInfo) {
+                //  iterate through storage columns
+                for(size_t storageColumnInfoIndex = 0; storageColumnInfoIndex < storageTableInfo.size();
+                    ++storageColumnInfoIndex) {
+
+                    //  get storage's column info
+                    auto& storageColumnInfo = storageTableInfo[storageColumnInfoIndex];
+                    auto& columnName = storageColumnInfo.name;
+                    if(storageColumnInfo.hidden) {
+                        columnsToAdd.push_back(&storageColumnInfo);
+                    }
+                }
+            }
             template<class I>
-            void backup_table(sqlite3* db, const I& tableImpl, const std::vector<table_info*>& columnsToIgnore) {
+            void drop_create_with_loss(const I& tImpl, sqlite3* db) {
+
+                try {
+                    this->start_migration();
+
+                    this->drop_table_internal(tImpl.table.name, db);
+                    this->create_table(db, tImpl.table.name, tImpl);
+
+                    this->commit_migration();
+                } catch(std::exception& ex) {
```

Comment:
Please catch by const-reference, and e.g. comment out the variable name.

Reply:
ok done